### PR TITLE
1

### PR DESCRIPTION
**Pull request by xyngabo**

### CODE REVIEW

**Comment by xyngabo** [comment]
Sweet. I will use this little program a lot.

**Comment by mmarchetti2** [comment]
Wait, what does this do on your computer? It seems you're the only one who got the check to pass, so I looked at your code, and it doesn't create any images. Does it work on yours?

**Comment by xyngabo** [comment]
yea, it worked on my pc, it made the images in the folder that are logoed.

**Comment by xyngabo** [comment]
It initially was giving me an error that seemed to be within my python build.

**Comment by xyngabo** [comment]
the book linked to stackoverflow for a tutorial to install pillow.. which linked to another program.. I just used: pip install Pillow

**Comment by mmarchetti2** [comment]
Huh, oh well. I still can't get mine to pass the checks, even though it works xD Thanks, though.

**Comment by halfaro1** [comment]
@xyngabo the program doesn't create files for me either. It is passing the automated tests because the submission already contains the resized files (maybe from a previous version you created on your computer?) but if you remove `withLogo` folder and run your code again it doesn't recreate the files with the logo. I'll comment inline to point you in the right direction but you'll need to keep working on this.

File: resizeAndAddLogo.py [hunk 1]
```diff
@@ -8,7 +8,23 @@
 logoWidth, logoHeight = logoIm.size
 
 #TODO: Loop over all files in the working directory
-#TODO: Check if file image needs to be
-#TODO: Calculate the new width and height to resize to.
-#TODO: Add the logo.
-#TODO: Save changes.
+os.makedirs('withLogo', exist_ok=True)
+
+for filename in os.listdir('.'):
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @xyngabo this line means that you're looking in the current working directory -- you might want to try looking in the `originals` directory

File: resizeAndAddLogo.py [hunk 2]
```diff
@@ -8,7 +8,23 @@
...
+    im = Image.open(filename)
+    width, height = im.size
+    if width > SQUARE_FIT_SIZE and height > SQUARE_FIT_SIZE:
+        if width > height:
+            height = int((SQUARE_FIT_SIZE / width) * height)
+            width = SQUARE_FIT_SIZE
+        else:
+            width = int((SQUARE_FIT_SIZE / height) * width)
+            height = SQUARE_FIT_SIZE
+    print('Resizing %s...' % (filename))
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @xyngabo shouldn't this be indented?

**Comment by halfaro1** [comment]
@xyngabo 
The automated testing for this particular assignment isn't working as expected, so I've graded your project manually. Everything is 👍 on your project, so I am closing this and changing the tag to `graded`. When I transfer these grades to Blackboard, you'll see the tag switch to `grade updated on blackboard`. Great job! (ps - you can still correct what I suggested in my last comment)

**Pull request closed without merging**